src/mpid/ch4/generic/am/mpidig_am.h

```diff
@@ -36,4 +36,6 @@ int MPIDIG_am_reg_cb_dynamic(MPIDIG_am_origin_cb origin_cb, MPIDIG_am_target_msg
 int MPIDIG_am_init(void);
 void MPIDIG_am_finalize(void);
 
+int MPIDIG_comm_abort(MPIR_Comm * comm, int exit_code);
```

Comment:
If we are going to make the move, we need rename it to `MPIDIG_am_comm_abort`

Reply:
Right, I will do that change last

Reply:
I am still unsure whether this should actually be in a `common` dir inside `generic`. This is not `am` specific.

Reply:
It is `am` specific. It is falling back to a generic routine using `am`. In principle, it is equally valid to falling back to a generic routine using `gpu`.

Reply:
Yes it is, sorry for the confusion.